### ISSUE TITLE
Some links are incorrect

### ISSUE THREAD

**Issue by topfunky** [hashicorp/terraform-website#607]
Several broken links were found on Terraform documentation pages.

```
Getting links from: https://www.terraform.io/docs/providers/aws/guides/eks-getting-started.html
├─BROKEN─ https://docs.aws.amazon.com/eks/latest/userguide/ (ERRNO_ENOTFOUND)
├─BROKEN─ https://docs.aws.amazon.com/eks/latest/userguide/ (HTTP_undefined)
├───OK─── https://kubernetes.io/
├───OK─── https://kubernetes.io/docs/tasks/tools/install-kubectl/
├───OK─── https://github.com/kubernetes-sigs/aws-iam-authenticator
├───OK─── https://docs.aws.amazon.com/cli/latest/reference/eks/update-kubeconfig.html
├─BROKEN─ https://docs.aws.amazon.com/eks/latest/userguide/ (HTTP_undefined)
├───OK─── https://github.com/terraform-providers/terraform-provider-aws/tree/master/examples/eks-getting-started
├───OK─── https://github.com/terraform-providers/terraform-provider-aws/edit/master/website/docs/guides/eks-getting-started.html.md
└───OK─── http://kubernetes.io/docs/user-guide/configmap/
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/artifacts
├─BROKEN─ https://packer.io/docs/post-processors/atlas.html#artifact_type (HTTP_404)
Finished! 116 links found. 115 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/packer/builds/troubleshooting.html
├───OK─── https://packer.io/docs/other/debugging.html
├─BROKEN─ https://www.packer.io/intro/getting-started/setup.html (HTTP_404)
├───OK─── https://github.com/mitchellh/packer
├───OK─── https://packer.io/
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/packer/builds/troubleshooting.html.md
└───OK─── http://groups.google.com/group/packer-tool
Finished! 131 links found. 125 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/packer/builds/build-environment.html
├───OK─── https://packer.io/docs/templates/provisioners.html
├─BROKEN─ https://packer.io/docs/command-line/push.html (HTTP_404)
└───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/packer/builds/build-environment.html.md
Finished! 148 links found. 145 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/packer/builds/starting.html
├───OK─── https://www.terraform.io/docs/enterprise-legacy/glossary#build-configuration
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/packer/builds/starting.html.md
├─BROKEN─ https://packer.io/docs/command-line/push.html (HTTP_404)
├─BROKEN─ https://packer.io/docs/templates/push.html#exclude (HTTP_404)
└─BROKEN─ https://packer.io/docs/templates/push.html#vcs (HTTP_404)
Finished! 121 links found. 116 excluded. 3 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/packer/artifacts/creating-vagrant-boxes.html
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/packer/artifacts/creating-vagrant-boxes.html.md
├───OK─── https://packer.io/docs/templates/post-processors.html
├───OK─── https://packer.io/docs/post-processors/vagrant.html
└─BROKEN─ https://packer.io/docs/post-processors/atlas.html (HTTP_404)
Finished! 129 links found. 125 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/packer/artifacts/index.html
├─BROKEN─ https://packer.io/docs/post-processors/atlas.html (HTTP_404)
├───OK─── https://www.terraform.io/docs/enterprise-legacy/
└───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/packer/artifacts/index.html.md
Finished! 129 links found. 126 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/vcs/bitbucket.html
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/vcs/bitbucket.html.md
├───OK─── https://bitbucket.org/account/user/your-username/oauth-consumers/new
├───OK─── https://bitbucket.org/account/user/your-username/api
├─BROKEN─ https://api.bitbucket.org/2.0 (HTTP_404)
└───OK─── https://bitbucket.org/your-username/your-repository/admin/access
Finished! 130 links found. 125 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/vcs/gitlab.html
├─BROKEN─ http://example.com/ (HTTP_404)
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/vcs/gitlab.html.md
├─BROKEN─ https://gitlab.com/api/v4 (HTTP_404)
├───OK─── https://atlas.hashicorp.com/settings
├───OK─── https://atlas.hashicorp.com/configurations/import
├───OK─── https://atlas.hashicorp.com/terraform/your-organization/environments/your-environment/changes/runs
├───OK─── https://atlas.hashicorp.com/builds/new
├───OK─── https://atlas.hashicorp.com/packer/your-organization/build-configurations/your-build-configuration
Finished! 129 links found. 121 excluded. 2 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise-legacy/artifacts/index.html
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise-legacy/artifacts/index.html.md
├───OK─── https://packer.io/docs
└─BROKEN─ https://packer.io/docs/post-processors/atlas.html#artifact_type (HTTP_404)
Finished! 116 links found. 113 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/enterprise/private/migrate.html
├───OK─── https://www.terraform.io/docs/enterprise/private/simplified-migration.html
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise/private/migrate.html.md
├─BROKEN─ https://console.aws.amazon.com/rds/home?region=us-east-1#dbinstances: (HTTP_401)
└─BROKEN─ https://console.aws.amazon.com/rds/home?region=us-east-1#db-snapshots: (HTTP_401)
Finished! 218 links found. 214 excluded. 2 broken.
```

```
Getting links from: https://www.terraform.io/docs/providers/heroku/index.html
├─BROKEN─ https://ec.haxx.se/usingcurl-netrc.html (HTTP_204)
```

```
Getting links from: https://www.terraform.io/docs/providers/gitlab/index.html
├─BROKEN─ https://my.gitlab.server/api/v3/ (ERRNO_ENOTFOUND)
```

```
Getting links from: https://www.terraform.io/docs/enterprise/guides/recommended-practices/part3.2.html
├───OK─── https://www.terraform.io/docs/enterprise/guides/recommended-practices/images/image2-12cb4459.png
├───OK─── https://www.vagrantup.com/docs/provisioning/chef_solo.html
├───OK─── https://docs.microsoft.com/en-us/azure/architecture/
├───OK─── https://github.com/mspnp/reference-architectures
├───OK─── https://d0.awsstatic.com/whitepapers/architecture/AWS_Well-Architected_Framework.pdf
├───OK─── https://www.amazon.com/Terraform-Running-Writing-Infrastructure-Code-ebook/dp/B06XKHGJHP/ref=sr_1_1?ie=UTF8&qid=1496138592&sr=8-1&keywords=terraform+up+and+running
├───OK─── http://www.intigua.com/blog/puppet-vs.-chef-vs.-ansible-vs.-saltstack
├───OK─── https://www.hashicorp.com/training/
├───OK─── https://github.com/hashicorp/terraform-website/edit/master/content/source/docs/enterprise/guides/recommended-practices/part3.2.html.md
├───OK─── https://docs.oracle.com/cloud/latest/stcomputecs/STCSG/GUID-C37FDFF1-7C48-4DA8-B31F-D7D7B35674A8.htm#STCSG-GUID-C37FDFF1-7C48-4DA8-B31F-D7D7B35674A8
├───OK─── https://www.amazon.com/Getting-Started-Terraform-Kirill-Shirinkin/dp/1786465108/ref=sr_1_1?ie=UTF8&qid=1496138892&sr=8-1&keywords=Getting+Started+with+Terraform
├─BROKEN─ https://www.drupalwatchdog.com/volume-4/issue-2/version-control-workflow-strategies (BLC_UNKNOWN)
```

```
Getting links from: https://www.terraform.io/docs/modules/sources.html
├───OK─── https://www.terraform.io/docs/registry/private.html
├───OK─── https://git-scm.com/docs/git-clone#_git_urls_a_id_urls_a
├───OK─── https://www.terraform.io/docs/registry/api.html
├───OK─── https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage
├───OK─── https://www.terraform.io/docs/registry/modules/use.html#using-modules
├───OK─── https://www.terraform.io/docs/enterprise/workspaces/ssh-keys.html
├───OK─── https://www.mercurial-scm.org/repo/hg/help/urls
├───OK─── https://github.com/hashicorp/terraform/edit/master/website/docs/modules/sources.html.markdown
├───OK─── http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#access-bucket-intro
└─BROKEN─ https://ec.haxx.se/usingcurl-netrc.html (HTTP_204)
Finished! 195 links found. 185 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/configuration/environment-variables.html
└─BROKEN─ https://github.com/hashicorp/terraform/edit/master/website/docs/configuration/environment-variables.html.md (HTTP_404)
Finished! 156 links found. 155 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/configuration/interpolation.html
├───OK─── https://www.terraform.io/docs/state/environments.html
├───OK─── https://www.terraform.io/docs/providers/template/index.html
├───OK─── https://www.terraform.io/docs/providers/template/d/file.html
├───OK─── https://golang.org/pkg/fmt/
├───OK─── https://github.com/google/re2/wiki/Syntax
└─BROKEN─ https://github.com/hashicorp/terraform/edit/master/website/docs/configuration/interpolation.html.md (HTTP_404)
Finished! 256 links found. 250 excluded. 1 broken.
```

```
Getting links from: https://www.terraform.io/docs/configuration/load.html
├───OK─── https://en.wikipedia.org/wiki/Declarative_programming
└─BROKEN─ https://github.com/hashicorp/terraform/edit/master/website/docs/configuration/load.html.md (HTTP_404)
Finished! 145 links found. 143 excluded. 1 broken.
```

**Comment by topfunky** [comment]
From the Packer team:

>We should remove references to Atlas and the `push` command (rather than fixing incorrect links).

**Comment by nfagerlund** [comment]
- Broken edit links don't need any action, they're fixed now that the 0.12 beta shipped.
- https://ec.haxx.se/usingcurl-netrc.html is fine, idk what that 204 was about.
- drupal watchdog thing, ugh, but I found the content at http://www.drupalwatchdog.net/volume-4/issue-2/version-control-workflow-strategies (has to be insecure http, their cert is way borked) https://github.com/hashicorp/terraform-website/pull/722
- gitlab thing, example url. markdown getting over-excited. https://github.com/terraform-providers/terraform-provider-gitlab/pull/108
- https://docs.aws.amazon.com/eks/latest/userguide/ is actually fine, idk what the error was about. 
- anything in /docs/enterprise-legacy/: who cares, let it rot
- aws rds instances link: it's fine, but it's a convenience link to part of your actual aws account, so it requires a login link. I think in this context that's probably ok.
